<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sioxtest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portTextbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>COM3</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>351</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="activatePort1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aktivera port 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deactivatePort1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deaktivera port 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
   <widget class="QLabel" name="busLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bus</string>
    </property>
   </widget>
   <widget class="QLabel" name="busNumberLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="commandLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Command</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandTextbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendCommand">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>230</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send command</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="speedTextbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9600</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
